internal/impl/ollama/embed_ollama_linux_arm64.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Licensed as a Redpanda Enterprise file under the Redpanda Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+// https://github.com/redpanda-data/connect/blob/main/licenses/rcl.md
+
+//go:build x_benthos_extra
```

Comment:
I re-used this existing extra, but I am happy to either remove the embedded server feature, or use a different tag